### ISSUE TITLE
Stride Guide?

### ISSUE THREAD

**Issue by NothingButQuestions** [brichard19/BitCrack#289]
Due to the nature and mechanism of action that stride (appears) to have on hex, is there not potentially a better or "more/most efficient" stride number? I have watched the the stride 'ticks' at different increments and its overall effect on the 64 hex. unlike standard numerical rollover, the stride function seems to effect blocks at different locations on the hex. I don't really know what I am talking about here tbh, but have a deep gut feeling about this based on the outputs i have checked using --continue and checking at 1 min intervals.

i am curious to better understand strides step by step effect on hex and also any smarter user's experiences/successes using specific [or ranges] of stride settings.

TL;DR- too small takes eons, too big misses everything...... what size should i set my stride porridge?

danke
